packages/camlp5/camlp5.8.00.01/opam

```diff
@@ -0,0 +1,59 @@
+name: "camlp5"
+version: "8.00.01"
+opam-version: "2.0"
+synopsis: "Preprocessor-pretty-printer of OCaml"
+description: """
+Camlp5 is a preprocessor and pretty-printer for OCaml programs. It also provides parsing and printing tools.
+
+As a preprocessor, it allows to:
+
+extend the syntax of OCaml,
+redefine the whole syntax of the language.
+As a pretty printer, it allows to:
+
+display OCaml programs in an elegant way,
+convert from one syntax to another,
+check the results of syntax extensions.
+Camlp5 also provides some parsing and pretty printing tools:
+
+extensible grammars
+extensible printers
+stream parsers and lexers
+pretty print module
+It works as a shell command and can also be used in the OCaml toplevel."""
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+authors: ["Daniel de Rauglaudre" "Chet Murthy"]
+license: "BSD-3-Clause"
+homepage: "https://camlp5.github.io"
+doc: "https://camlp5.github.io/doc/html"
+bug-reports: "https://github.com/camlp5/camlp5/issues"
+depends: [
+  "ocaml" {>= "4.02" & < "4.13.0"}
```

Comment:
I think that it should me `>= "4.10"`
@chetmurthy

Reply:
um, camlp5 8.00 should work fine with OCaml 4.02 -- it just won't have any of the new syntax available.

Reply:
I'll go further: it should be a -bug- if Camlp5 8.xx does not work with Ocaml >= 4.02 .